Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Maybe we can remove this file, since we don't rely on SwiftHash any more?

Reply:
It's an auto-generated file by `swift package update`. I just re-ran it to remove the SwiftHash